### DISCUSSION TITLE
Version 23.2.3.202310221533. Does not auto fill double quotes for column name

### DISCUSSION THREAD

**Discussion by lnm1956** [dbeaver/dbeaver#21640] [DBeaver Development]
try to create query something like this
select * from "ClientData" cd  where cd."Id" = 1  

in previous versions auto fill add cd."Id" but now -  cd.Id 
result:
SQL Error [42703]: ERROR: column cd.id does not exist
 Hint: Perhaps you meant to reference the column "cd.Id".

**Comment by LonwoLonwo** [comment]
Hello @lnm1956 

We already have a ticket for this: https://github.com/dbeaver/dbeaver/issues/21589.
You can track its history.
